### ISSUE TITLE
Command Line on Windows Systems?

### ISSUE THREAD

**Issue by Lifeling** [dwservice/agent#27]
I don't see anything on the website that says that this feature is not available for windows. However, when I look to access a windows agent, this option is not available.

**Comment by BGrewell** [comment]
It is commented out because it is old code and things have changed since that code was written. I have submitted a pull request with the changes needed to support Windows terminal, although it will only support Windows 10 devices as Windows 7 doesn't support a unix type pty.

**Comment by Lifeling** [comment]
@BGrewell Thanks for that. I do hope they can bring this feature back, it would be very useful. Are you able to build and run a client based on your code?

**Comment by BGrewell** [comment]
Yes, all of the code to enable the terminal is client side, you technically
don’t even need to build it you can just copy the app_shell directory and
the applications.py file into your install directory after you install the
standard agent and it will work since the applications.py file tells the
browser what applications are supported.
